helpers/channels/threads/getArchivedThreads.ts

```diff
@@ -42,14 +39,14 @@ export async function getArchivedThreads(
         return [member.id, member];
       }),
     ),
+    hasMore: result.has_more,
   };
 }
 
-// TODO: add docs link
-export interface ListPublicArchivedThreads {
-  // TODO: convert unix to ISO9601 timestamp
-  /** Returns threads before this timestamp. UNIX or ISO8601 timestamp */
-  before?: number | string;
+/** https://discord.com/developers/docs/resources/channel#list-public-archived-threads-query-string-params */
+export interface ListArchivedThreads {
+  /** Returns threads before this timestamp */
+  before?: number;
```

Comment:
Has this been tested?

Reply:
My bad I forgot to convert it to ISO string